src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -105,11 +107,28 @@ void tabulate() throws TabulationCancelledException {
     Logger.log(Level.INFO, "Starting tabulation session...");
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
     if (config != null && config.validate() && setUpLogging(config)) {
+      try {
+        Logger.log(Level.INFO, "Begin config file contents:");
+        BufferedReader reader = new BufferedReader(new FileReader(configPath));
+        String line = reader.readLine();
+        while(line != null) {
+          Logger.log(Level.INFO, line);
+          line = reader.readLine();
+        }
+        Logger.log(Level.INFO, "End config file contents.");
+        reader.close();
+      } catch (IOException e) {
+        Logger.log(Level.SEVERE, "Error logging config file: %s\n", configPath, e.toString());
+      }
+      Logger.log(Level.INFO, "Tabulating \'%s\'...", config.getContestName());
       if (config.isSequentialMultiSeatEnabled()) {
+        Logger.log(Level.INFO, "This is a sequential multi-seat contest.");
         int numWinners = config.getNumberOfWinners();
         // temporarily set config to single-seat so we can run sequential elections
         config.setNumberOfWinners(1);
         while (config.getSequentialWinners().size() < numWinners) {
+          Logger.log(Level.INFO, "Beginning sequence: %d...",
```

Comment:
"Beginning tabulation for seat #%d..." would be clearer for me.

Reply:
Yeah that's good idea